{"name": "Проверка ошибки при попытке обновления данных неавторизованного пользователя", "status": "passed", "description": "Тест проверяет, что неавторизованный пользователь не может обновить имя и получает ошибку", "steps": [{"name": "Изменить данные пользователя: email, password, name", "status": "passed", "parameters": [{"name": "access_token", "value": "''"}, {"name": "new_data", "value": "{'name': 'new_name_WoW'}"}], "start": 1718404938350, "stop": 1718404938907}], "start": 1718404938349, "stop": 1718404938909, "uuid": "0f125386-75cf-4cee-82b7-88bdbf213bdf", "historyId": "805abfbda1639ba49e9d487fa38d4c18", "testCaseId": "805abfbda1639ba49e9d487fa38d4c18", "fullName": "tests.test_change_user_data.TestChangeUserData#test_user_cant_update_name_without_authorization", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_user_data"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "DESKTOP-212965U"}, {"name": "thread", "value": "15752-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_user_data"}]}